experimental/transformers-to-ctranslate/main.py

```diff
@@ -0,0 +1,61 @@
+import os
+from args import make_parser
+
+from dotenv import load_dotenv
+from ctranslate2.converters.transformers import TransformersConverter
+from huggingface_hub import snapshot_download
+from transformers.convert_slow_tokenizers_checkpoints_to_fast import convert_slow_checkpoint_to_fast
+from huggingface_hub import login, HfApi
+
+if __name__ == "__main__":
+    # Set up args
+    parser = make_parser()
+
+    args = parser.parse_args()
+
+    # Load env
+    load_dotenv()
+
+    # Check out model
+    model_path = snapshot_download(
+        repo_id=args.model,
+        cache_dir=args.output_dir,
+        force_download=True
+    )
+    convert_output_dir = os.path.join(model_path, "ctranslate2")
+
+    # Convert model into ctranslate
+    converter = TransformersConverter(
+        args.model,
+        activation_scales=args.activation_scales,
+        copy_files=args.copy_files,
+        load_as_float16=args.quantization in ("float16", "int8_float16"),
```

Comment:
is `quantization` defined in argparser?

Reply:
It is declared in https://github.com/TabbyML/tabby/blob/converter/experimental/transformers-to-ctranslate/args.py#L54, which is defined in https://github.com/OpenNMT/CTranslate2/blob/master/python/ctranslate2/converters/converter.py#L15

Reply:
but you don't define it in arg parser right? so it will be `None`?

Neitherless, let's pass reasonble default to transformer converter, as long as the output file is stored as `float16`